lib/ansiblelint/utils.py

```diff
@@ -483,7 +511,9 @@ def get_action_tasks(yaml, file):
     block_rescue_always = ('block', 'rescue', 'always')
     tasks[:] = [task for task in tasks if all(k not in task for k in block_rescue_always)]
 
-    return [task for task in tasks if 'include' not in task.keys()]
+    return [task for task in tasks if
+            ('include' or 'include_tasks' or 'import_tasks')
```

Comment:
```
{'include', 'include_tasks', 'import_tasks'}.isdisjoint(task.keys())
```
might be simpler, and actually work:

```
>>> ('d' or 'b') not in v
True
>>> {'d', 'b'}.isdisjoint(v)
False
```

Reply:
I'd like there to be a test that would have caught this bug

Reply:
What bug exactly? 
The wrong return for get_action_tasks?